scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -887,7 +939,7 @@ s32 pick_idle_cpu(struct task_struct *p, s32 prev_cpu,
 			cpu = -1;
 			goto out_put;
 		}
-		if ((cpu = pick_idle_cpu_from(cpumask, prev_cpu, idle_smtmask)) >= 0)
+		if ((cpu = pick_idle_cpu_from(cpumask, prev_cpu, cast_mask(idle_smtmask))) >= 0)
```

Comment:
Ditto. Here, `cpumask` contains the CPUs that are owned by this layer in the previous LLC. Let's say the layer is a protected grouped one; then, `idle_smtmask` from idle_smtmask_unprotected() wouldn't contain any CPUs which intersect with the owned CPUs.